massgen/frontend/displays/textual_terminal_display.py

```
@@ -10525,6 +10542,31 @@ def action_toggle_theme(self) -> None:
             self.notify(f"Theme: {new_theme.title()}", timeout=1.5)
 
+        def action_toggle_copy_mode(self) -> None:
+            """Toggle copy mode (Ctrl+Shift+S).
+
+            Releases the terminal mouse so the user can drag-select text natively
+            and copy with the terminal's built-in shortcut. Press again to restore
+            Textual's normal mouse behavior.
+            """
+            try:
+                banner = self.query_one(CopyModeBanner)
+            except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)